saleor/graphql/app/filters.py

```diff
@@ -3,13 +3,12 @@
 from ...app import models
 from ...app.types import AppExtensionTarget, AppType
 from ..core.filters import EnumFilter, ListObjectTypeFilter
-from ..utils.filters import filter_by_query_param
 from .enums import AppExtensionMountEnum, AppExtensionTargetEnum, AppTypeEnum
 
 
 def filter_app_search(qs, _, value):
     if value:
-        qs = filter_by_query_param(qs, value, ("name",))
+        qs = qs.filter(name__ilike=value)
```

Comment:
Previous solution was also applying `return queryset.filter(query_objects).distinct()` to the queryset. Does `.distinct` is not needed here?

Reply:
Nope, it was a generic func that was putting distinct to everything. Here we have a simple filter with no joins, so no distinct is needed.